src/rules/no-misleading-boolean-prefixes.ts

```
@@ -0,0 +1,332 @@
+             items: { type: 'string' },
+           },
+         },
+         additionalProperties: false,
+       },
+     ],
+     messages: {
+       nonBooleanReturn:
+         'Function "{{name}}" uses a boolean-style prefix but does not return a boolean value.',
+     },
+   },
+   defaultOptions: [{ prefixes: DEFAULT_PREFIXES }],
```

ESLint

[error] Delete `·`

(prettier/prettier)